database/factories/GenreFactory.php

```diff
@@ -18,7 +20,10 @@ public function definition(): array
             "mal_id" => $mal_id,
             "name" => $name,
             "url" => $url,
-            "count" => $this->faker->randomDigit()
+            "count" => $this->faker->randomDigit(),
+            "createdAt" => new UTCDateTime(),
+            "modifiedAt" => new UTCDateTime(),
+            "request_hash" => sprintf("request:%s:%s", "v4", sha1("http://localhost-test/v4/genres/". $this->mediaType ."/" . $mal_id))
```

Comment:
Here as well
`sha1("http://localhost-test/v4/genres/". $this->mediaType ."/" . $mal_id)`